internal/bminventory/inventory.go

```diff
@@ -2787,31 +2790,35 @@ func (b *bareMetalInventory) calculateHostNetworks(log logrus.FieldLogger, clust
 func (b *bareMetalInventory) listClustersInternal(ctx context.Context, params installer.V2ListClustersParams) ([]*models.Cluster, error) {
 	log := logutil.FromContext(ctx, b.log)
 	db := b.db
+
+	var dbClusters []*common.Cluster
+	var clusters []*models.Cluster
+
+	//this call should be before OwnedBy
+	if params.Owner != nil {
+		ctxparams.SetParam(ctx, auth.X_USER_SCOPE, params.Owner)
+	}
+
 	if swag.BoolValue(params.GetUnregisteredClusters) {
+		//bypass for the GC mechanism
```

Comment:
GUC?:)

Reply:
Garbage Collection (This is part of the api that servers only the GC server)